### DISCUSSION TITLE
Error_scf_

### DISCUSSION THREAD

**Discussion by callas7** [nguyen-group/QERaman#22] [Q&A] [answered]
Hello Prof. Nguyen, 
Thank you for the code (QERaman). It is very helpful. I tried to run the calculation for a nanotube system, but I got the following error. 
<img width="521" alt="error_Raman" src="https://github.com/user-attachments/assets/bca468df-bd09-4e91-918a-7a640c378d06">
[scf.docx](https://github.com/user-attachments/files/17428870/scf.docx)
[output_scf.docx](https://github.com/user-attachments/files/17428872/output_scf.docx)

**Comment by nguyen-group** [comment]
Hi, 

If you used a, b, c in input scf.in, you should not use ibrav =0 and CELL_PARAMETERS.
I suggest removing the CELL_PARAMETERS block and changing ibrav = 8.

Best,
Nguyen

**Reply by callas7** [reply]
I used CELL_PARAMETTERS block after relaxing my system. Won't it affect the relaxation of the system dear professor?

**Comment by callas7** [comment]
I made the changed you told me Professor, I still face the same error

**Reply by nguyen-group** [reply, marked as answer]
Hi, could you remove "assume_isolated = '2D'" in scf.in too, it is for only 2D system, but your system is 1D.

Best,
Nguyen
